src/sentry/event_manager.py

```diff
@@ -46,6 +47,7 @@
 from sentry.stacktraces import normalize_in_app
 
 
+HASH_RE = re.compile(r'^[0-9a-f]{32}$')
```

Comment:
case insensitive?

Reply:
Good call, is JSON-schema always case insensitive? If not I think we need to adjust the Snuba schema too.